astropy/vo/client/tests/test_conesearch.py

```diff
@@ -149,6 +149,9 @@ def test_async(self):
         async_search = conesearch.AsyncConeSearch(
             SCS_CENTER, SCS_RADIUS, pedantic=self.pedantic)
 
+        # Wait a little for the instance to set up properly
+        time.sleep(3)
```

Comment:
Asynchronous function is meant to run in the background. It is certainly will not be done in 3 seconds. Waiting for 3 seconds simply allow the next line (`.get()`) to function properly. It is that line that will raise timeout error if it takes too long. I don't know what it is needed now and not before (it is not like this test had not passed in the past). Hopefully this is all that is to it, as I am still waiting for the test result from Travis CI.